pkg/apis/pipeline/v1beta1/result_types.go

```diff
@@ -25,7 +25,7 @@ type TaskResult struct {
 
 	// Description is a human-readable description of the result
 	// +optional
-	Description string `json:"description"`
+	Description string `json:"description,omitempty"`
```

Comment:
@lbernick I check the code before https://github.com/tektoncd/pipeline/pull/4779 and it didn't contain `omitempty`, do you think we should add it or not?